libraries/AP_Mount/AP_Mount.h

```diff
@@ -188,12 +192,18 @@ class AP_Mount
 
     // get mount's current attitude in euler angles in degrees.  yaw angle is in body-frame
     // returns true on success
+    bool get_attitude_euler(float& roll_deg, float& pitch_deg, float& yaw_bf_deg) { return get_attitude_euler(_primary, roll_deg, pitch_deg, yaw_bf_deg); }
     bool get_attitude_euler(uint8_t instance, float& roll_deg, float& pitch_deg, float& yaw_bf_deg);
 
     // run pre-arm check.  returns false on failure and fills in failure_msg
     // any failure_msg returned will not include a prefix
     bool pre_arm_checks(char *failure_msg, uint8_t failure_msg_len);
 
+    // get mount's target angles in degrees. yaw angle is in body-frame
+    // returns true on success
+    bool get_target_attitude_euler(float& roll_deg, float& pitch_deg, float& yaw_deg) { return get_target_attitude_euler(_primary, roll_deg, pitch_deg, yaw_deg); }
```

Comment:
I don't actually see why we need this new method, "get_target_attitude_euler", when we have a nearly identical, "get_angle_target" just below.  I think we should try and re-use the existing method.